admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -933,6 +933,21 @@ var form = (function() {
         send($('.product-footer .go-catalog').attr('data-redirect'));
       });
 
+      /** create keyboard event for save & go preview */
+      jwerty.key('ctrl+V', function(e) {
+          e.preventDefault();
+          send($('.product-footer .preview').attr('data-redirect'), $('.product-footer .preview').attr('target'));
```

Comment:
Why not ? : (ping other, i'm not sure)
`var productFooter = $('.product-footer .preview');`
`send(productFooter.attr('data-redirect'), productFooter.attr('target'));`